model/service.go

```diff
@@ -77,6 +77,11 @@ type Port struct {
 
 	// Protocol to be used for the port.
 	Protocol Protocol `json:"protocol,omitempty"`
+
+	// In combine with the mesh's AuthPolicy, controls authentication for
+	// Envoy-to-Envoy communication.
+	// This value is extracted from service annotation.
+	SecurityOption SecurityOption `json:"security_option,omitempty"`
```

Comment:
Do we need an option here? A mesh has either opt-in or opt-out.. no point in having this for per port. I suggest using a simple boolean  like `Secure bool`.

Reply:
I think we do. The point of this is to overwrite the (global) mesh's policy. A service can have more than one port, and we want to have a fine grained control for each of them.

Reply:
Also, boolean may not be enough. We want a 'state' that indicates to use mesh's policy to decide. Furthermore, it's possible to add other options in future (e.g regular TLS instead of mTLS, etc)

Reply:
Make sense. Then just change it to AuthenticationPolicy